app/forms/questionnaire_form.py

```diff
@@ -176,11 +176,24 @@ def _get_target_total_and_currency(
         question: QuestionSchema,
     ) -> Optional[tuple[Union[Calculation, AnswerValueTypes], Optional[str]]]:
 
+        value_source_resolver = ValueSourceResolver(
+            answer_store=self.answer_store,
+            schema=self.schema,
+            metadata=self.metadata,
+            response_metadata=self.response_metadata,
+            list_store=self.list_store,
+            location=self.location,
+            list_item_id=None,
```

Comment:
This needs to be set to work within repeats.

Reply:
Ok added.